CHANGELOG.md

```diff
@@ -31,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Access log level was changed from `INFO` to `DEBUG` to avoid logging a message for every poll by each client.
   To restore the old behavior, start wfx with `--log-level=DEBUG` (note that this will enable additional log messages
   though).
+- `wfxctl`: return non-zero exit code if `job events` connection is lost
```

Comment:
The reason for this is that you get notified of connection loss and can then check wfx's job view to determine whether you lost events or not, see above. In some scenarios losing events may be fine, in others not so that you can exercise control with this changed behavior.

Reply:
Done